srcpkgs/python-pywatchman/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'python-pywatchman'
+pkgname=python-pywatchman
+version=1.4.1
+revision=1
+wrksrc="pywatchman-${version}"
+build_style=python-module
+only_for_archs="x86_64 x86_64-musl"
+pycompile_module="pywatchman"
+hostmakedepends="python-devel python3-devel python-setuptools python3-setuptools"
+makedepends="curl"
+depends="python watchman"
+short_desc="Connect and query Watchman to discover file changes (Python 2)"
+maintainer="Hoang Nguyen <hoang@wetrust.io>"
+license="BSD"
+homepage="https://facebook.github.io/watchman/"
+distfiles="${PYPI_SITE}/p/pywatchman/pywatchman-${version}.tar.gz"
+checksum=d0047eb275deafb0011eda0a1a815fbd9742478c3d2b5ad6956d300e447dc2f9
+alternatives="watchman-make:watchman-make:/usr/bin/watchman-make2 watchman-wait:watchman-wait:/usr/bin/watchman-wait2"
+
+post_install() {
+	curl https://raw.githubusercontent.com/facebook/watchman/master/python/LICENSE > LICENSE
+	vlicense LICENSE
+}
+
+python3-pywatchman_package() {
+	short_desc="${short_desc/2/3}"
+	depends="python3 watchman"
+	alternatives="watchman-make:watchman-make:/usr/bin/watchman-make3 watchman-wait:watchman-wait:/usr/bin/watchman-wait3"
```

Comment:
multiple alternatives invocations can be made on multiple lines

alternatives="
    foo:foo:usr/bin/foo
    bar:bar:/usr/bin/bar"